firebase-app/src/androidMain/kotlin/dev/gitlive/firebase/firebase.kt

```diff
@@ -29,11 +29,11 @@ actual fun Firebase.initialize(context: Any?, options: FirebaseOptions, name: St
 actual fun Firebase.initialize(context: Any?, options: FirebaseOptions) =
     FirebaseApp(com.google.firebase.FirebaseApp.initializeApp(context as Context, options.toAndroid()))
 
-actual class FirebaseApp internal constructor(val android: com.google.firebase.FirebaseApp) {
+actual data class FirebaseApp internal constructor(val android: com.google.firebase.FirebaseApp) {
```

Comment:
Why a data class now?

Reply:
So you can do proper equality checks. Useful for testing